tool/gocross/gocross-wrapper.sh

```diff
@@ -8,7 +8,7 @@
 
 set -euo pipefail
 
-if [[ "${CI:-}" == "true" ]]; then
+if [[ "${CI:-}" == "true" && "${NOBASHDEBUG:-}" != "true" ]]; then
```

Comment:
I had to do this to make courtney happy, as it executes `go` and can't handle the bash debug output from `set -x`.